internal/utils/nat.go

```diff
@@ -119,6 +208,16 @@ func (c *Client) CreateNat(datacenterId, name string, publicIps []string, lansTo
 		OpenPort("ALL", 0)              // Outbound
 	rules := nrm.Make()
 
+	new_rules, err := natRulesStringToModel(natRules, publicIps[0])
+
+	if err != nil {
+		return nil, err
+	}
+
+	if new_rules != nil {
+		*rules = append(*rules, *new_rules.Items...)
+	}
```

Comment:
I think, if the user specifies some custom nat rules, then it should override the default ones, instead of adding to them?